src/main/java/edu/kis/powp/jobs2d/commands/ComplexCommandFactory.java

```diff
@@ -0,0 +1,47 @@
+package edu.kis.powp.jobs2d.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class ComplexCommandFactory {
+
+    private int startX;
+    private int startY;
+
+    public ComplexCommandFactory(int x, int y){
+        this.startX = x;
+        this.startY = y;
+    }
```

Comment:
Creating a factory instance fixed on specific starting coordinates is not a very convincing idea.